manager-bundle/src/Composer/ScriptHandler.php

```diff
@@ -34,6 +34,8 @@ public static function initializeApplication(Event $event): void
         static::executeCommand(sprintf('assets:install %s --symlink --relative', $webDir), $event);
         static::executeCommand(sprintf('contao:install %s', $webDir), $event);
         static::executeCommand(sprintf('contao:symlinks %s', $webDir), $event);
+
+        $event->getIO()->write('<info>Done! Please open the install tool and make sure the database is up-to-date.</info>');
```

Comment:
Should we perhaps add an additional blank line below so that the info message is more visible?